NuKeeper/ProcessRunner/UnixProcess.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Diagnostics;
+using System.Threading.Tasks;
+
+namespace NuKeeper.ProcessRunner
+{
+    public class UnixProcess : IExternalProcess
```

Comment:
At this stage IMHO a better set of names would be `WindowsExternalProcess` and `UnitExternalProcess`

Reply:
Agreed.

Reply:
Er, I mean `UnixExternalProcess`.